HRIS.Services/Services/FieldCodeOptionsService.cs

```diff
@@ -49,6 +70,14 @@ public async Task<List<FieldCodeOptionsDto>> GetAllFieldCodeOptions()
 
     public async Task<List<FieldCodeOptionsDto>> UpdateFieldCodeOptions(List<FieldCodeOptionsDto> fieldCodeOptionsDto)
     {
+        var fieldCodeOptionExist = await FieldCodeOptionsExists(fieldCodeOptionsDto[0].FieldCodeId);
+
+        if (!fieldCodeOptionExist)
+            throw new CustomException("Field Code Option does not exist");
+
+        if (_identity.IsSupport == false && fieldCodeOptionsDto[0].FieldCodeId != _identity.EmployeeId)
```

Comment:
`fieldCodeOptionsDto[0].FieldCodeId != _identity.EmployeeId`

Is `FieldCodeId` the `EmployeeId` ?